### PR TITLE
Updated selectors documentation for element ID

### PR DESCRIPTION
**Pull request by reds71**
Moved the "_id_" section that described a deprecated feature to the end and said it has to been done using regular CSS or XPath selectors (syntax given). Could also have purely removed it ?

### CODE REVIEW

File: docs/Selectors.md [hunk 1]
```diff
@@ -147,6 +131,9 @@ You can use xPath to also traverse up and down the DOM tree, e.g.
 const parent = paragraph.$('..');
 console.log(parent.getTagName()); // outputs: "body"
 ```
+## id
+
+Finding element by id has no specific syntax in WebDriver and one should use either CSS selectors (`#<my element ID>`) or xPath (`//*[@id="<my element ID>"]`).
```
Comment (christian-bromann) [inline docs/Selectors.md]:
```suggestion
Finding element by id has no specific syntax in WebDriver and one should use either CSS selectors (`#<my element ID>`) or xPath (`//*[@id="<my element ID>"]`). However some drivers (e.g. [Appium You.i Engine Driver](https://github.com/YOU-i-Labs/appium-youiengine-driver#selector-strategies)) might still [support](https://github.com/YOU-i-Labs/appium-youiengine-driver#selector-strategies) this selector.
```

**Comment by christian-bromann** [comment]
ping @reds71

**Comment by christian-bromann** [comment]
any interest in this @reds71 ?

File: docs/Selectors.md (hunk 1 above)
Comment (reds71) [reply docs/Selectors.md]: Sorry Christian. Did not understand that you were waiting for me and thought the ping was to initiate a french translation (for which I'm neither qualified nor available). As you may have noticed, I'm new to WebdriverIO, Web development & GitHub... So I fully trust you for you amendment !

**Comment by reds71** [comment]
Is it OK? Do I have something else to do to unleash the merge ? If you are interested in feedback from a newbie that previously tried Protractor and was close to turn back away from WebdriverIO, tell me on what channel. Cheers.

**Comment by christian-bromann** [comment]
> Do I have something else to do to unleash the merge ?

I was suggesting a small change in order to mention that the id selector strategy might be suppose in drivers e.g. You.i TV driver. I am still interested in merging this as long as we can add this mention to it.

**Comment by reds71** [comment]
I saw you suggestion and agreed but I have not the rights to "commit suggestion". How should I proceed ? Report it "by hand" in my clone copy and create a new PR (or it will be automatic) ? Sounds complicated since you are almighty in your project and I said OK... I really need an update on Github workflow!

**Comment by christian-bromann** [comment]
@reds71 just make another push on this branch in your GitHub fork

**Pull request merged**